ansys/mapdl/core/convert.py

```diff
@@ -270,17 +393,47 @@ def __init__(
         self._block_current_cmd = None
 
     def write_header(self):
-        header = f'"""Script generated by ansys-mapdl-core version {__version__}"""'
-        self.lines.append(header)
+        if isinstance(self._header, bool):
+            if self._header:
+                header = f'"""Script generated by ansys-mapdl-core version {__version__}"""'
+                self.lines.append(header)
+        elif isinstance(self._header, str):
+            self.lines.append(f'"""{self._header}"""')
+
+        else:
+            raise TypeError("The keyword argument 'header' should be a string or a boolean.")
 
     def write_exit(self):
         self.lines.append(f"{self.obj_name}.exit()")
 
-    def save(self, filename):
+    def format_using_autopep8(self, text=None):
```

Comment:
```suggestion
    def format_using_autopep8(self, text=None):
        """Format internal `self.lines` with autopep8.

        Parameters
        ----------
        text : str, optional
            Text to format instead of `self.lines`. For development use.

        """
```